app/controllers/invitations_controller.rb

```diff
@@ -1,22 +1,22 @@
 class InvitationsController < Devise::InvitationsController
+  # Copied from https://github.com/scambra/devise_invitable/blob/master/app/controllers/devise/invitations_controller.rb
+  # And edited. This is a common devise pattern, similar to OmniauthCallbacksController.
+
+  # PUT /resource/invitation
   def update
-    # Copied from https://github.com/scambra/devise_invitable/blob/master/app/controllers/devise/invitations_controller.rb
-    # And edited. This is a common devise pattern, similar to OmniauthCallbacksController.
     raw_invitation_token = update_resource_params[:invitation_token]
     self.resource = accept_resource
-    invitation_accepted = resource.errors.empty?
```

Comment:
This is needed and was accidentally removed -- I have since added this back to the code and will push the change up in a moment!